packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -1453,6 +1453,48 @@ module.exports = (
         return {};
       },
     },
+    {
+      method: 'GET',
+      path: '/account',
```

Comment:
@philbooth Sorry if I missed the reasoning, but why wouldn't we want to hook onto the [/account/profile](https://github.com/mozilla/fxa/blob/14314844e4c8b74d3f151a39809fe04c4009d518/packages/fxa-auth-server/lib/routes/account.js#L932) endpoint? Is it because that route is a high traffic route and used by profile server? 

Only mentioning because it appears to return subscription data already and is sessionToken authenticated.

Reply:
> why wouldn't we want to hook onto the /account/profile endpoint?

Fwiw I didn't consider it at the time because it's not one of the endpoints currently requested by the settings screen and I didn't want to weigh down other callers with stuff they don't need.

Looking at it now though, the subscriptions data returned from it is different to what I need, it's product ids from the auth db. I specifically want the `failure_code` property from subhub for #970 and we also want the `plan_name` property from subhub for #1801. Neither of those is in the auth db (by design), so we'd need to change its behaviour to make the extra request to subhub and also the behaviour of its consumer(s) so they don't expect strings.

I'm minded not to interfere with the other consumer like that and stick to using a fresh endpoint, but also happy to make the change if you think that's wrong. Let me know!

Reply:
> but also happy to make the change if you think that's wrong

Nope, your reasoning sounds good. We have a couple endpoints that do account status related things and it wouldn't mind deprecate some of them in favor of this route.

Side note, looks like you will need to update the docs for route.

Reply:
> need to update the docs for route

Ah dammit. I remember when those docs used to update themselves, sigh.